nilearn/reporting/_get_clusters_table.py

```diff
@@ -131,7 +131,7 @@ def _get_val(row, input_arr):
 
 
 def get_clusters_table(stat_img, stat_threshold, cluster_threshold=None,
-                       min_distance=8.):
+                       two_sided=True, min_distance=8.):
```

Comment:
I think I'd default to False for the sake of bw compatibility.

Reply:
Good point.